server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -46,7 +46,7 @@
 
     public static final Version NUMERIC_PAGINATION_VERSION = Version.V_7_15_0;
 
-    private static final Version SORT_BY_SHARD_COUNTS_VERSION = Version.V_7_16_0;
+    private static final Version SORT_BY_SHARDS_OR_REPO_VERSION = Version.V_7_16_0;
```

Comment:
Being a little lazy here :) <= we don't run BwC tests that would fail because of this and this is going to backport problem free.